Cargo.lock

```diff
@@ -1557,9 +1557,9 @@ dependencies = [
 
 [[package]]
 name = "crate_crypto_internal_eth_kzg_bls12_381"
-version = "0.5.1"
+version = "0.5.2"
```

Comment:
Ok I think this is the issue. Idk why `cargo update` is updating this when we have a precise version constraint in the `Cargo.toml` introduced in https://github.com/sigp/lighthouse/pull/6608/files.